packages/db-migrations/databases/fxa/patches/patch-123-124.sql

```diff
@@ -0,0 +1,10 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('123');
+
+ALTER TABLE accounts
```

Comment:
We had discussed adding the googleId/etc to the email table along with the email/verification status the google Id verified, was it more compelling to leave it on the accounts table? If we store this on the account table, and then also store the Apple Auth on the main account table, does that mean we're intentionally prohibiting account merging with dual login (Google and Apple) and the merge must choose the valid auth scheme? I think its fine to make that a requirement of merging, though if that does happen, I can imagine people accidentally creating accounts again by clicking the wrong one.... needing yet more merging.

Reply:
> was it more compelling to leave it on the accounts table

I believe so. I think the biggest issue was the FxA allows users to change their email address and that would cause some extra complexity here.

Instead of adding to the accounts table, we could create a new table, `providers` and store `uid`, `id`, `provider`, `authAt` properties?